tensorrt_llm/_torch/pyexecutor/cuda_graph_runner.py

```
@@ -28,58 +29,60 @@ class CUDAGraphRunner:
+    def __init__(
+        self,
+        *,
+        use_cuda_graph: bool,
+        cuda_graph_padding_enabled: bool,
+        supported_batch_sizes: list[int],
+        max_supported_batch_size: int,
+        max_batch_size: int,
+        max_beam_width: int,
+        max_draft_len: int,
+        use_mrope: bool,
+        spec_config: Optional["DecodingBaseConfig"],
```

Ruff (0.12.2)

Undefined name `DecodingBaseConfig`

(F821)